201-representing-CNAB-in-OCI.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Representing CNAB bundles in OCI Registries
+weight: 201
+---
+
+# Representing CNAB bundles in OCI Registries
+
+This section describes how the bundle file and the information represented in the bundle MUST be represented when distributing bundles using OCI registries.
+
+A CNAB is made up of a descriptor file (`bundle.json`), one or more invocation images, and a list of zero or more component images. Each of these objects can be individually packaged into an [OCI manifest][oci-manifest] or [OCI index][oci-index]. An OCI index can then be used to combine these all into a single object.
```

Comment:
Is "object" the most accurate term we have for that? Perhaps "artifact", "referenceable artifact" or something else? If that's the term from the OCI spec, ignore me. 😀 It just doesn't seem to convey much meaning and is a bit overloaded.

Reply:
I would prefer not to use "artifact", as it has a specific meaning in the context of the [OCI artifacts](https://github.com/opencontainers/artifacts) project.